tiatoolbox/models/architecture/hovernetplus.py

```diff
@@ -34,22 +34,61 @@
 
 
 class HoVerNetPlus(HoVerNet):
-    """Initialise HoVerNet+.
+    """Initialise HoVerNet+ [1].
 
     HoVerNet+ takes an RGB input image, and provides the option to simultaneously
     segment and classify the nuclei present, aswell as semantically segment different
     regions or layers in the images. Note the HoVerNet+ architecture assumes an image
     resolution of 0.5 mpp, in contrast to HoVerNet at 0.25 mpp.
 
+    The tiatoolbox model should produce following results on the private OED dataset
```

Comment:
```suggestion
    The tiatoolbox model should produce following results on the specified datasets
```

In future we may have more data sets.